lib/rubocop/cop/performance/big_decimal_with_numeric_argument.rb

```diff
@@ -3,45 +3,48 @@
 module RuboCop
   module Cop
     module Performance
-      # Identifies places where numeric argument to BigDecimal should be
-      # converted to string. Initializing from String is faster
-      # than from Numeric for BigDecimal.
+      # Identifies places where string argument to `BigDecimal` should be
+      # converted to numeric. Initializing from Integer is faster
+      # than from String for BigDecimal.
       #
       # @example
       #   # bad
-      #   BigDecimal(1, 2)
-      #   4.to_d(6)
-      #   BigDecimal(1.2, 3, exception: true)
-      #   4.5.to_d(6, exception: true)
-      #
-      #   # good
       #   BigDecimal('1', 2)
       #   BigDecimal('4', 6)
       #   BigDecimal('1.2', 3, exception: true)
       #   BigDecimal('4.5', 6, exception: true)
       #
+      #   # good
+      #   BigDecimal(1, 2)
+      #   4.to_d(6)
+      #   BigDecimal(1.2, 3, exception: true)
+      #   4.5.to_d(6, exception: true)
+      #
       class BigDecimalWithNumericArgument < Base
         extend AutoCorrector
+        extend TargetRubyVersion
+
+        minimum_target_ruby_version 3.1
```

Comment:
Can you use `requires_gem` API here? bigdecimal is becoming a bundled gem so many already have it in their lockfile and the ruby version is barely relevant anymore. This would stop this cop from doing something if the user doesn't use bundler but I think this is the way to go

Reply:
Ah, right. You already meant `requires_gem` wit with that